website/source/docs/command-line/build.html.md

```diff
@@ -36,7 +36,17 @@ artifacts that are created will be outputted at the end of the build.
     remove the artifacts from the previous build. This will allow the user to
     repeat a build without having to manually clean these artifacts beforehand.
 
+-   `-on-error=cleanup` (default), `-on-error=abort`, `-on-error=ask` - Selects
+    what to do when the build fails.  `cleanup` cleans up after the previous
+    steps, deleting temporary files and virtual machines.  `abort` exits without
+    any cleanup, but the next build may require `-force`.  `ask` presents a
```

Comment:
> `abort` exits without any cleanup, but the next build may require `-force`.

Maybe rephrase to:

`abort` exits without any cleanup, which might require the next build to use `-force`.